documentation/modules/rn-2.6.adoc

```diff
@@ -58,9 +58,25 @@ The {project-short} web console can no longer download logs. With this update, y
 
 This part provides features and enhancements introduced in {project-full} 2.6:
 
+.Preservation of a static IP in a migration from vSphere
+
+You can now preserve a static IP when importing VMs that have vSphere as their source provider. link:https://issuesredhat.com/browse/MTV-1148[(MTV-1148)]
+
+.Entering a list of LUKS decryption passphrases in a vSphere migration plan
+
+You can now enter a single list of multiple Linux Unified Key Setup (LUKS) decryption passphrases when you create a vSphere migration plan, rather than having to enter them one-by-one. For each LUKS-encrypted device, {project-short} tries each passphrase until one unlocks the device. link:https://issuesredhat.com/browse/MTV-831[(MTV-831)]
+
+.Specifying the primary disk when migrating from vSphere
+
+You can now specify the primary disk when you migrate VMs from vSphere which have both a primary and backup disk. This avoids {project-short} attempting to update the backup disk instead of the primary disk. link:https://issuesredhat.com/browse/MTV-1079[(MTV-1079)]
```

Comment:
yeah, the term "primary" can be confusing here - it's not about primary vs. backup, but about which of the bootable disks needs to be considered as the disk that the VM would boot from in OpenShift (the conversion process makes adjustments only to a single operating system)
```suggestion
You can now specify the primary disk when you migrate VMs from vSphere which have more than a single bootable disk. This avoids {project-short} attempting to convert the first bootable disk that is detected while examining the disks that is not necessarily the disk that the VM is expected to boot from in OpenShift. link:https://issuesredhat.com/browse/MTV-1079[(MTV-1079)]
```